slugs.md

```diff
@@ -1359,6 +1359,7 @@ update the script at 'scripts/build-slugs-table.js'.
 | `Qwiklabs` | `qwiklabs` |
 | `Qzone` | `qzone` |
 | `R` | `r` |
+| `R3` | `r3` |
```

Comment:
I _think_ this file is auto-generated with our weekly releases, so editing it isn't necessary. Could you please revert this file?

Reply:
I have reverted the change.